mypy/test/data/check-overloading.test

```diff
@@ -678,3 +678,16 @@ def f(x: int) -> None: pass
 def f(x: B) -> str: pass # E: Overloaded function signatures 2 and 3 overlap with incompatible return types
 @overload
 def f(x: A) -> int: pass
+
+[case testOverloadTuple]
+from typing import overload, Tuple
+@overload
+def f(x: int, y: Tuple[str, ...]) -> None: pass
+@overload
+def f(x: int, y: str) -> None: pass
+f(1, ('2', '3'))
```

Comment:
Also test calling with a `Tuple[str, ...]` arg and a `Tuple[int, ...]` argument.